### DISCUSSION TITLE
Redirect from the Pages Domain

### DISCUSSION THREAD

**Discussion by Rummelbude** [community/community#102865] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey,

I own a GitHub Repository with GitHub Pages configured. The URL of a GitHub Pages site is username.github.io/repository. I want, that if I open username.github.io, without specifying the repository, it redirects me to username.github.io/repository. Is there a way to configure that or can I add an index.html directly to my GitHub profile without a repository?

Thanks,
Rummelbude

**Comment by queenofcorgis** [comment]
Hi there @Rummelbude, 

Are you asking if it is possible to create a GitHub Pages site without a repository? That would not be possible as GitHub Pages site must be created in a new or existing repository.

**Comment by Rummelbude** [comment, marked as answer]
I actually managed it, but with creating a repository. You can just create a repository and call it username.github.io and it works as I wanted.
